packages/claimed-credential/schemas/ClaimedCredential.json

```diff
@@ -0,0 +1,71 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "ClaimedCredential",
+  "description": "A [[ClaimedCredential]] represents a specific, factually-oriented claim that could be the Id and CredentialSubject of a [[VerifiableCredential]]",
+  "type": "object",
+  "properties": {
+    "did": {
+      "type": "string"
+    },
+    "type": {
+      "type": "string"
+    },
+    "typeSchema": {
+      "type": "string"
+    },
+    "value": {
+      "type": "object"
```

Comment:
This should specify supported properties, no?

Reply:
Added supported properties